README.md

```diff
@@ -66,19 +66,18 @@ To learn more about the different rule types, their guarantees and how they are
 #### Carrying out software supply chain steps
 
 ##### in-toto-run
-`in-toto-run` generates link metadata for the given command-line option and runs it as its own command. See the [this simple usage example from the demo application for more details](https://github.com/in-toto/demo#tampering-with-the-software-supply-chain).
 
-```shell
-in-toto-run  --step-name <unique step name>
-            {--key <functionary signing key path>,  --gpg [<functionary gpg signing key id>]}
-            [--gpg-home <path to gpg keyring>]
-            [--materials <filepath>[ <filepath> ...]]
-            [--products <filepath>[ <filepath> ...]]
-            [--record-streams]
-            [--no-command]
-            [--verbose] -- <cmd> [args]
-```
+`in-toto-run` is used to execute one of the software supply chain steps. The 
+relevant step name and command are passed as arguments, along with materials, 
+which are files required for command execution, and products which are files
+expected as a result of the execution of the command. These, and other 
+relevant details pertaining to the step are stored in a link file, which is 
+signed using the functionary's key.
```

Comment:
I think it could still be made a little clearer that `in-toto-run` should be used to executes any command corresponding to a step of the software supply chain, such as tagging a software release with git, running tests or building a binaries.

Also the explanation about materials might be a little confusing. Which command requires them for execution? `in-toto-run` or the passed command? And will execution fail if they are not passed? (*It does not, but the materials are simply not recorded in the corresponding link file, if not passed*).